eventsourcing/src/test/java/org/axonframework/eventsourcing/eventstore/AggregateBasedStorageEngineTestSuite.java

```diff
@@ -261,24 +264,14 @@ void concurrentTransactionsForNonOverlappingTagsBothCommit()
                                          taggedEventMessage("event-0", OTHER_AGGREGATE_CRITERIA.tags()))
                            .get(1, TimeUnit.SECONDS);
 
-        CompletableFuture<Long> firstCommit = firstTx.commit();
-        CompletableFuture<Long> secondCommit = secondTx.commit();
+        CompletableFuture<ConsistencyMarker> firstCommit = firstTx.commit();
+        CompletableFuture<ConsistencyMarker> secondCommit = secondTx.commit();
 
         assertDoesNotThrow(() -> firstCommit.get(1, TimeUnit.SECONDS));
         assertDoesNotThrow(() -> secondCommit.get(1, TimeUnit.SECONDS));
 
-        assertEquals(0, firstCommit.join());
-        assertEquals(0, secondCommit.join());
-    }
-
-    @Test
-    public void shouldRejectAppendCriteriaWithMoreThanOneIndex() {
```

Comment:
I triggered a little on the removal of this test (yet again). Left a comment with more explanation on the `LegacyAxonServerEventStorageEngine` for that.

Reply:
I removed this because there is no reason all implementations should reject more than one tag on the criteria. They should refuse more than one tag on an event, as that's not supported with aggregates.